ui/pages/Login/components/Login.jsx

```diff
@@ -21,6 +22,8 @@ const Login = ({ onSubmit }) =>
     submitButtonText="Log In"
   >
     <Link to="/users/forgot_password">Forgot Password?</Link>
+    <Divider />
+    <a href="/login/google-oauth2">Sign in with Google</a>
```

Comment:
We should only show this button if we have social auth enabled in seqr. So in `react_app.py` we render the page with some initial json, and we should update that something like
```
initial_json['meta']['googleLoginEnabled'] = bool(TERRA_API_ROOT_URL)
```
Then you can add a selector that checks `state.meta.googleLoginEnabled` and make this button conditional on that

Reply:
Done